Documentation/Sources/templates/basic.md

```diff
@@ -74,7 +74,9 @@ The tables below contain guidelines for your push notification content. These va
 
 ### Text guidelines
 
-| **Type** | **Characters per line** | **Recommended max<br /># of characters** | **Supports multiple<br />lines?** |
-| :-------- | -----------: | ------: | -------: |
-| Title | 35-40 | 35 | No |
-| Description | 50-60 | 250 (~5 lines) | Yes |
+| **Type** |**Recommended max<br /># of characters** | **Supports multiple<br />lines?** |
+| :-------- | ------: | -------: |
+| Title | 70 | Yes (2 lines max) |
+| Description | 600 | Yes |
+
```

Comment:
Simplified this table a little bit.
Removed column - `Characters per line`. It wont be useful, considering the template rendering on iPad portrait will have a very high number of characters per line.

I think the useful recommendation for our customers would be the maximum recommended character size for the whole text.
In this case, it is derived as the maximum number of characters rendered in the smallest device iPhone mini

Title  - 35 x 2 lines = 70 
Description - 30 x 15 lines (maximum) = 600